prime-router/metadata/hl7_mapping/ORU_R01/base/message-header.yml

```diff
@@ -84,7 +84,9 @@ elements:
     hl7Spec: [ MSH-6-3 ]
 
   - name: message-date-time
-    value: [ 'Bundle.entry.resource.ofType(Provenance).recorded' ]
+    value:
+      - Bundle.entry.resource.ofType(Provenance).recorded
+      - Bundle.timestamp
```

Comment:
I spoke to @carlosfelix2 regarding this change. `Bundle.timestamp` does not correctly store the timezone data, so `Bundle.entry.resource.ofType(Provenance).recorded` is preferred but `Bundle.timestamp` makes a good backup option if the preferred data is not so this is ultimately a good change.